x-pack/packages/kbn-ai-assistant/src/context/ai_assistant_app_service_provider.tsx

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createContext } from 'react';
+import type { AIAssistantAppService } from '../service/create_app_service';
+
+export const AIAssistantAppServiceContext = createContext<AIAssistantAppService | undefined>(
+  undefined
+);
```

Comment:
I think exporting contexts from packages is risky, because every time this package is included in a bundle, `AIAssistantAppServiceContext` gets its own identity. Are there any consequences that we should be worried (or not worried about)?